docs: Add YugabyteDB smart driver features to Sequelize documentation

krishna-yb opened the pull request:
- Added note about smart driver features in Step 2
- New section: Using YugabyteDB Smart Driver features
  - Load balancing configuration and modes
  - Topology awareness for multi-region deployments
  - Connection string examples with smart driver parameters
  - Environment variables reference table
- Examples showing how to enable load balancing with process.env
- Documentation for all PGLOADBALANCE modes
- Examples for topology-aware routing
- Complete environment variables reference

krishna-yb commented:
https://deploy-preview-29496--infallible-bardeen-164bc9.netlify.app/stable/develop/drivers-orms/nodejs/sequelize/


Sequelize deployment page

krishna-yb commented:
Latest sequelize page:
https://deploy-preview-29496--infallible-bardeen-164bc9.netlify.app/stable/develop/drivers-orms/nodejs/sequelize/

HarshDaryani896 reviewed (approved):
LGTM.

aishwarya24 commented on docs/content/stable/develop/drivers-orms/nodejs/sequelize.md:
```diff
@@ -88,6 +88,29 @@ The following code creates an Employees model to store and retrieve employee inf
 - The actual table is created by calling the `Employee.sync()` API in the `createTableAndInsert()` function. This also inserts the data for three employees into the table using the `Employee.create()` API.
 - Finally, you can retrieve the information of all employees using `Employee.findAll()`.
 
+#### Using the YugabyteDB Smart Driver
```
```suggestion
#### Using the YugabyteDB node-postgres smart driver
```

aishwarya24 commented on docs/content/stable/develop/drivers-orms/nodejs/sequelize.md:
```diff
@@ -88,6 +88,29 @@ The following code creates an Employees model to store and retrieve employee inf
+
+You can also use the YugabyteDB smart driver (`@yugabytedb/pg`) which provides connection load balancing feature that spreads connections uniformly across the nodes in the entire cluster or in specific placements (zones or regions).
+
+Read more on the smart driver features [here](../../smart-drivers/#using-yugabytedb-smart-drivers).
```
```suggestion
For more information on the smart driver features, refer to [Using YugabyteDB smart drivers](../../smart-drivers/#using-yugabytedb-smart-drivers).
```

aishwarya24 commented on docs/content/stable/develop/drivers-orms/nodejs/sequelize.md:
```diff
@@ -88,6 +88,29 @@ The following code creates an Employees model to store and retrieve employee inf
+
+Read more on the smart driver features [here](../../smart-drivers/#using-yugabytedb-smart-drivers).
+
+To use the YugabyteDB smart driver instead of the vanilla PG driver, add the following to your application's `package.json` dependencies:
```
```suggestion
To use the YugabyteDB node-postgres smart driver instead of the vanilla pg driver, add the following dependencies to your application's `package.json` file, to override the default PostgreSQL driver with the YugabyteDB smart driver:
```

aishwarya24 commented on docs/content/stable/develop/drivers-orms/nodejs/sequelize.md:
````diff
@@ -189,6 +212,59 @@ Employees Details:
+   dialect: 'postgres'
+})
+```
+
````
```suggestion
For information on all load balance modes, see [Node type-aware load balancing](../../smart-drivers/#node-type-aware-load-balancing).
```

aishwarya24 reviewed (commented):
some comments.

krishna-yb commented:
Hey @aishwarya24 
Please take a look
I've addressed your comments

Most recent sequelize page: [link](https://deploy-preview-29496--infallible-bardeen-164bc9.netlify.app/stable/develop/drivers-orms/nodejs/sequelize/)

ddhodge commented on docs/content/stable/develop/drivers-orms/nodejs/sequelize.md:
```diff
@@ -189,6 +210,71 @@ Employees Details:
+
+##### Environment variables
+
+The following table summarizes the environment variables available for configuring the YugabyteDB node-postgres smart driver:
```
@ashetkar These Env variables are common to the driver not just sequelize? Why is this table not here https://docs.yugabyte.com/stable/develop/drivers-orms/nodejs/yugabyte-node-driver/#step-2-set-up-the-database-connection
